presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improve performance of :func:`SUM` and :func:`AVG` aggregate functions when used with ``DECIMAL`` type.
+* Add column comment to metadata in JDBC based connector
+* Add hidden column ``$file_modified_time`` which is the time the file containing the row was last modified.
+* Add hidden column ``$file_size`` which is the size of the file containing the row.
+* Add LocationProvider support to Iceberg connector so files are now written to paths defined by Iceberg's configured LocationProvider instead of a hard-coded table root directory.
```

Comment:
Fix this as well based on the changes made below

Reply:
Oops, TY!